src/components/Amount.tsx

```diff
@@ -84,10 +89,14 @@ const Amount = ({
   const [integralPart, fractionalPart] = amount.split('.')
 
   return (
-    <span className={className} tabIndex={tabIndex ?? -1}>
-      {discreetMode ? (
-        '•••'
-      ) : value !== undefined ? (
+    <AmountStyled
+      {...{ className, color, value, highlight, tabIndex: tabIndex ?? -1, discreetMode }}
```

Comment:
Note the spreading technique. This makes it DRYier when passing props down to the styled component. I've added this proposition to our Notion documentation. We're not forced to use this if you don't like it obviously!